vsts/build-test-nuget.yml

```diff
@@ -299,11 +299,6 @@ jobs:
           command: "login"
           containerRegistry: "Azure IoT ACR"
 
-      - task: DockerInstaller@0
```

Comment:
This is the part caused the conflicts on the script. No longer need to install this as it is already configured in the latest agent image.